Demo/Demo/SwiftUIComponents/CardVaultViews/CreatePaymentTokenView.swift

```diff
@@ -0,0 +1,56 @@
+import SwiftUI
+
+struct CreatePaymentTokenView: View {
+
+    let selectedMerchantIntegration: MerchantIntegration
+    let setupToken: String
+
+    @ObservedObject var cardVaultViewModel: CardVaultViewModel
+
+    public init(cardVaultViewModel: CardVaultViewModel, selectedMerchantIntegration: MerchantIntegration, setupToken: String) {
+        self.cardVaultViewModel = cardVaultViewModel
+        self.selectedMerchantIntegration = selectedMerchantIntegration
+        self.setupToken = setupToken
+    }
+
+    var body: some View {
+        VStack(spacing: 16) {
+            HStack {
+                Text("Create a Permanent Payment Method Token")
+                    .font(.system(size: 20))
+                Spacer()
+            }
+            .frame(maxWidth: .infinity)
+            .font(.headline)
+            ZStack {
+                if case .loading = cardVaultViewModel.state.paymentTokenResponse {
+                    ProgressView()
+                        .progressViewStyle(CircularProgressViewStyle(tint: Color.white))
+                        .background(Color.black.opacity(0.4))
+                        .cornerRadius(10)
+                        .frame(maxWidth: .infinity)
+                }
+                Button("Create Payment Token") {
+                    Task {
+                        do {
+                            try await cardVaultViewModel.getPaymentToken(
+                                setupToken: setupToken,
+                                selectedMerchantIntegration: selectedMerchantIntegration
+                            )
+                        } catch {
+                            print("Error in getting setup token. \(error.localizedDescription)")
+                        }
+                    }
+                }
+            }
+            .foregroundColor(.white)
+            .padding()
+            .frame(maxWidth: .infinity)
+            .background(.blue)
+            .cornerRadius(10)
+        }
+        .padding()
+        .background(RoundedRectangle(cornerRadius: 10).stroke(Color.gray, lineWidth: 2))
+        .padding(5)
```

Comment:
Do we need the `padding` modifier twice here?

Reply:
I'm still uncertain where these paddings take effect but from putting color red on the VStack, I saw that the first modifier puts the default padding of 10 between the content in VStack and outline of the Stack itself, whereas the second padding adds a little space between the rounded outline and the main view outline.